docs/source/api_user.rst

```diff
@@ -7,9 +7,7 @@ These are reference class and function definitions likely to be useful to everyo
    intake.open_catalog
    intake.registry
    intake.register_driver
-   intake.unregister_driver
```

Comment:
intake.source.discovery.unregister_driver ; should be intake.unregister_driver, but that points incorrectly to intake.source.unregister_driver.